tests/data/test_fetch_fpn_data.py

```diff
@@ -40,4 +40,4 @@ def test_fetch_fpn_data():
         )
 
     assert len(df_boa) == 215
-    assert len(df_fpn) == 3010
+    assert len(df_fpn) == 2926
```

Comment:
im slightly confused why this number went down? but other tests stayed the same so i feel ok by this